int32 im_width
int32 im_height
int32[] im_data
float32[] tracks_data
int32 tracks_width
int32 tracks_height
